### ISSUE TITLE
Add probas to StructuredDataClassifier

### ISSUE THREAD

**Issue by alexcombessie** [keras-team/autokeras#890]
Right now in 1.0.0b only hard predictions are output

**Comment by alexcombessie** [comment]
Still interested by the way, any chance it can happen in a later version?

**Comment by haifeng-jin** [comment]
I think this is totally doable with a small trick. Just export the model and get the output of the softmax.

**Comment by alexcombessie** [comment]
@haifeng-jin I agree, it's not difficult to code it with this trick. As it's a common requirement, I think it would be valuable to have it built-in the core library.

**Comment by alexcombessie** [comment]
Still very interested by this feature!
